### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by mcleary** [lucas-clemente/language-cmake#5]
Store package settings files in the `settings/` directory instead of `scoped-properties/`
```
Package.loadSettings (/Applications/Atom.app/Contents/Resources/app/src/package.js:550:9)
<unknown> (/Applications/Atom.app/Contents/Resources/app/src/package.js:172:43)
```

**Comment by lucas-clemente** [comment]
Fixed by #4

**Comment by mcleary** [comment]
I know atom if a Github project but I love this system to post an issue in a package with one click.
